modules/reference/pages/systems/systems-list.adoc

```diff
@@ -58,7 +58,8 @@ The menu:Systems[System List > Requiring Reboot] section lists every client that
 
 The menu:Systems[System List > Non Compliant] section lists every client that has non-compliant packages installed.
 Packages are considered non-compliant if they have been installed from a source other than {productname}.
-In the list, the [guimenu]``Packages`` column shows the number of non-compliant packages on the client.
+In the list, the [guimenu]``Extra Packages`` column shows the number of non-compliant packages on the client.
+These numbers are links to listings of those non-compliant packages per system.
```

Comment:
Cool.

Beware: if the number is zero, there's no link, just the number

add ", unless there are no such packages." ?